web/app.js

```diff
@@ -2061,6 +2063,22 @@ const PDFViewerApplication = {
     return wholeTicks;
   },
 
+  _accumulateFactor(previousScale, factor, prop) {
+    if (factor === 1) {
+      return 1;
+    }
+    // If the direction changed, reset the accumulated ticks.
```

Comment:
Too much copy-and-paste in the comment :-)
```suggestion
    // If the direction changed, reset the accumulated factor.
```